saleor/graphql/checkout/schema.py

```diff
@@ -101,3 +102,5 @@ class CheckoutMutations(graphene.ObjectType):
     )
     checkout_delivery_method_update = CheckoutDeliveryMethodUpdate.Field()
     checkout_language_code_update = CheckoutLanguageCodeUpdate.Field()
+
+    order_from_checkout_create = OrderFromCheckoutCreate.Field()
```

Comment:
Maybe `orderCreateFromCheckout`? It follows the pattern `object_action` - the action here is "creating from checkout" and the object is "order", not "orderFromCheckout". Again I won't fight about this but it sounds better to me, you can discuss with the checkout team if it is worth changing or they like the current name.